ending.gd

```diff
@@ -0,0 +1,42 @@
+extends Node2D
+
+
+# If true, NPCs spawn left side of the room in ending. False means right side.
+# This will be alternated on every spawn.
+var spawn_left = true
+# List of possible [scale, y-position] for random y adjustment in NPC spawns.
+var random_scale_y = [[0.085, 502], [0.082, 497], [0.08, 492]]
+
+
+func _ready():
+	Global.is_ending = true
+	$PartyProps/Banner/Label.text = Global.get_ending_text()
+	$MagicMonster.appear()
+	randomize()
+	spawn_npcs()
+
+
+# Return to opening scene on interact.
+func _input(event):
+	if event.is_action_pressed("interact") or event.is_action_pressed("jump"):
+		Transition.wipe_in(preload("res://ui/opening.tscn"))
```

Comment:
if user accidentally click interact during ending scene it will just throw user to opening screen?

Reply:
Yeah, just pressing Interact (Space) once will return to title screen. Jump is also included because controller uses another button for jump (it's a convention thing) instead of the directionals, but because this is not differentiated with the keyboard controls, "W" key will also trigger it. This is the current behavior of the jam submission version, so this PR did not change that.

Perhaps it would be good to use another key, or show a prompt so that the player has to press Interact twice or something. Since this PR is just to separate the ending through refactoring, and does not cover changing its behavior, I opened issue #27.